spec/controllers/state_file/questions/eligible_controller_spec.rb

```diff
@@ -17,5 +17,27 @@
       expect(response).to be_successful
       expect(response_html).to have_text I18n.t("state_file.questions.eligible.edit.title1", year: MultiTenantService.statefile.current_tax_year, state: "Arizona")
     end
+
+    context "AZ" do
+      let(:intake) { create :state_file_az_intake }
+  
+      it "does not show NJ content, links to AZ FAQ" do
+        get :edit
+        expect(response.body.html_safe).to include I18n.t('state_file.questions.eligible.vita_option.want_to_claim_learn_more_html', link: "/en/az/faq/other_state_filing_options")
+        expect(response.body.html_safe).not_to have_text ActionController::Base.helpers.strip_tags(I18n.t('state_file.questions.eligible.vita_option.vita_eligibility_reveal.nj')).delete("n")
+        expect(response.body.html_safe).not_to have_text ActionController::Base.helpers.strip_tags(I18n.t('state_file.questions.eligible.vita_option.connect_to_vita.nj')).delete("n")
+      end
+    end
+
+    context "NJ" do
+      let(:intake) { create :state_file_nj_intake }
+  
+      it "shows vita_eligibility_reveal content and not connect_to_vita content" do
+        get :edit
+        expect(response.body.html_safe).to include I18n.t('state_file.questions.eligible.vita_option.vita_eligibility_reveal.nj', vita_link_href: "https://airtable.com/appqG5OGbTLBiQ408/pag9EUHzAZzfRIwUn/form")
+        expect(response.body.html_safe).to include I18n.t('state_file.questions.eligible.vita_option.want_to_claim_learn_more_html', link: "/en/nj/faq/other_filing_options")
+        expect(response.body.html_safe).not_to have_text ActionController::Base.helpers.strip_tags(I18n.t('state_file.questions.eligible.vita_option.connect_to_vita.nj')).delete("n")
```

Comment:
[pebble] i don't think this check is really necessary? outside of the context of this PR it doesn't really make sense